### DISCUSSION TITLE
想问您，利用Quicker查词，您展示的那个gif中pdf软件是什么

### DISCUSSION THREAD

**Discussion by huiguo07** [crimx/ext-saladict#1489] [Q&A] [answered]
## 设备信息
- 操作系统: [] 
- 浏览器版本: [] 
- 沙拉查词版本: []

**Comment by crimx** [comment]
我也不清楚，@hqyyqh 大佬，你的本地 PDF 阅读器是什么来着？

**Comment by hqyyqh** [comment, marked as answer]
[pdf xchange editor](https://www.tracker-software.com/product/pdf-xchange-editor)，我用的[免费便携版](https://www.tracker-software.com/product/pdf-xchange-editor/download?fileid=694)
